walkthroughs/tls-with-acm/src/colorteller/Gopkg.toml

```diff
@@ -0,0 +1,30 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[prune]
```

Comment:
Given we're using dep, should we gitignore the vendor directory? A thought that I'm fine with being overridden on.

https://github.com/golang/dep/blob/master/docs/FAQ.md#should-i-commit-my-vendor-directory

Reply:
I think as a follow-up we can remove it. The dependency list is fairly small so I'm not terribly worried about reproducability of builds for these, and would prefer a cleaner repo myself.